components/Dashboard/DashboardContainer.js

```diff
@@ -1,36 +1,131 @@
-import RightBar from '../RightBar'
-import Typography from '@material-ui/core/Typography'
-import DashboardCard from "./Cards/DashboardCard";
+import { React, useState } from 'react';
+import {
+  Container,
+  Box,
+  Grid,
+  Typography,
+  Paper,
+  useTheme,
+} from '@material-ui/core';
+import DashboardCard from './Cards/DashboardCard';
+import SpaceCardModal from './Cards/SpaceCardModal';
+import SpaceCard from './Cards/SpaceCard';
+
+import { spaceCardModalTestData } from '../../data/spaceCardModalTestData';
+import { spaceCardTestData } from '../../data/spaceCardTestData';
 
 const DashboardContainer = () => {
+  const { friends, participants, hosts } = spaceCardModalTestData;
+  const { cardData } = spaceCardTestData;
+  const [open, setOpen] = useState(false);
+  const theme = useTheme();
   return (
-    <div className="h-full">
-      <div className="px-8 py-1">
-        <Typography variant="h4" className="pt-4">
-          Hey Charles!
-        </Typography>
-        <Typography variant="h7">
-          The key is not to prioritize what's on your schedule, but to schedule
-          your priorities.
-        </Typography>
-      </div>
-      <div className="flex ml-3 mt-6 space-x-6 mr-4"></div>
+    <Container>
+      <Paper elevatation={6} className="rounded-3xl">
+        <Grid
+          container
+          direction="column"
+          alignContent="center"
+          alignItems="center"
+          justify="center"
+          spacing={6}>
+          <Grid container item xs={12}>
+            <Grid item xs={12}>
+              <Box color={theme.palette.primary.dark}>
+                <Typography variant="h4">Hey Charles!</Typography>
+              </Box>
+            </Grid>
+            <Grid item xs={12}>
+              <Box color={theme.palette.text.secondary}>
+                <Typography variant="h6">
+                  The key is not to prioritize what's on your schedule, but to
+                  schedule your priorities.
+                </Typography>
+              </Box>
+            </Grid>
+          </Grid>
+
+          {/* Dashboard Card Section */}
+
+          <Grid
+            container
+            direction="row"
+            item
+            alignContent="center"
+            alignItems="center"
+            justify="space-between"
+            spacing={3}
+            xs={12}>
+            <Grid item xs={6} container direction="column" spacing={3}>
+              <Grid item xs={12}>
+                <Typography variant="h6" color="textSecondary">
+                  Need a space to study?
+                </Typography>
+              </Grid>
+              <Grid item xs={12}>
+                <DashboardCard
+                  variant="dark"
+                  spaceName="Create a Space"
+                  description="insert some sort of tagline or feature description "
+                />
+              </Grid>
+            </Grid>
+            <Grid item xs={6} container direction="column" spacing={3}>
```

Comment:
I think this is the culprit for poor responsiveness. We are using `Grid` but were not taking full advantage of the flexibility it offers. Particularly with the "column width" props ie `xs` , `sm`, `md`. 
Web grid columns contain 12 fractions. We usually want adjacent grids to sum to 12. Common design pattern would be `xs={12} md={6}` which basically means "this grid col should be full width on x-small to medium and half width on medium and up"

![image](https://user-images.githubusercontent.com/68363001/123503170-454df800-d61f-11eb-93bb-93522ee42b30.png)